metadata-ingestion/tests/unit/sql_parsing/aggregator_goldens/test_table_swap_with_temp.json

```diff
@@ -0,0 +1,242 @@
+                    },
+                    "created": {
+                        "time": 0,
+                        "actor": "urn:li:corpuser:_ingestion"
+                    },
+                    "dataset": "urn:li:dataset:(urn:li:dataPlatform:snowflake,dev.public.foo,PROD)",
+                    "type": "TRANSFORMED",
+                    "query": "urn:li:query:composite_4d5c4eb337c2cfd3f538794297115259dfff9f3745ccf9c59ad8d2f1d537dfe4"
+                },
+                {
+                    "auditStamp": {
+                        "time": 1707182625000,
+                        "actor": "urn:li:corpuser:_ingestion"
+                    },
+                    "created": {
+                        "time": 0,
+                        "actor": "urn:li:corpuser:_ingestion"
+                    },
+                    "dataset": "urn:li:dataset:(urn:li:dataPlatform:snowflake,dev.public.foo_dep,PROD)",
+                    "type": "TRANSFORMED",
+                    "query": "urn:li:query:composite_4d5c4eb337c2cfd3f538794297115259dfff9f3745ccf9c59ad8d2f1d537dfe4"
+                }
+            ],
+            "fineGrainedLineages": [
+                {
+                    "upstreamType": "FIELD_SET",
+                    "upstreams": [
+                        "urn:li:schemaField:(urn:li:dataset:(urn:li:dataPlatform:snowflake,dev.public.foo,PROD),a)"
+                    ],
+                    "downstreamType": "FIELD",
+                    "downstreams": [
+                        "urn:li:schemaField:(urn:li:dataset:(urn:li:dataPlatform:snowflake,dev.public.foo_backup,PROD),a)"
+                    ],
+                    "confidenceScore": 0.2,
+                    "query": "urn:li:query:composite_4d5c4eb337c2cfd3f538794297115259dfff9f3745ccf9c59ad8d2f1d537dfe4"
+                },
+                {
+                    "upstreamType": "FIELD_SET",
+                    "upstreams": [
+                        "urn:li:schemaField:(urn:li:dataset:(urn:li:dataPlatform:snowflake,dev.public.foo_dep,PROD),b)"
+                    ],
+                    "downstreamType": "FIELD",
+                    "downstreams": [
+                        "urn:li:schemaField:(urn:li:dataset:(urn:li:dataPlatform:snowflake,dev.public.foo_backup,PROD),b)"
+                    ],
+                    "confidenceScore": 0.2,
+                    "query": "urn:li:query:composite_4d5c4eb337c2cfd3f538794297115259dfff9f3745ccf9c59ad8d2f1d537dfe4"
+                }
+            ]
+        }
+    }
+},
+{
+    "entityType": "query",
+    "entityUrn": "urn:li:query:composite_4d5c4eb337c2cfd3f538794297115259dfff9f3745ccf9c59ad8d2f1d537dfe4",
+    "changeType": "UPSERT",
+    "aspectName": "queryProperties",
+    "aspect": {
+        "json": {
+            "statement": {
+                "value": "CREATE TABLE foo_staging AS\nSELECT\n  f.a,\n  d.b\nFROM foo AS f\nJOIN foo_dep AS d\n  USING (c);\n\nCREATE TABLE foo_backup AS\nSELECT\n  *\nFROM foo_staging",
```

Comment:
probably would be tricky to do, but it'd be super cool if the swap statement would show up in the composite query